src/publisher/Publisher.ts

```diff
@@ -122,6 +129,16 @@ export class Publisher extends OTEventEmitter<{
       })
       .on("left-meeting", () => {
         removeAllParticipantMedias();
+      })
+      .once("participant-updated", (dailyEvent) => {
+        // Fire local only once.
+        if (!dailyEvent?.participant.local) return;
```

Comment:
A couple of questions here:

* What happens if `"participant-updated"` does not fire?
* What happens if the first update is for a remote participant and we early out here, never listening for it again?

Basically, does the current logic support the code under this never running?